config/config.go

```diff
@@ -153,9 +153,9 @@ func (b *nullableBool) UnmarshalJSON(data []byte) error {
 	if err = json.Unmarshal(data, &v); err != nil {
 		return err
 	}
-	switch v.(type) {
+	switch v := v.(type) {
```

Comment:
Maybe `raw := v.(type)` if you prefer?
The new `v` variable shadows the previous one with a different type, this is not good.